### ISSUE TITLE
False positives when running with `--stdin-display-name`

### ISSUE THREAD

**Issue by bryanforbes** [plinss/flake8-noqa#24]
Given the following file (`blah.py`):

```python
print(
    'Lorem ipsum lorem ipsum Lorem ipsum lorem ipsum Lorem ipsum lorem ipsum Lorem ipsum lorem ipsum '  # noqa: E501
)
```

And the following `.flake8`:

```ini
[flake8]
max-line-length = 80
extend-select = NQA1
noqa-require-code = True
```

The following invocations result in two different outcomes:

```sh
$ flake8 blah.py


$ cat blah.py | flake8 --stdin-display-name blah.py -
blah.py:2:105: NQA102 "# noqa: E501" has no matching violations
```

**Comment by plinss** [comment]
It's actually failing on any input from stdin, easy fix. Thanks for the report. Fixed in v1.3.1, up on PyPI shortly.
